storage/testdb/testdb.go

```diff
@@ -39,21 +39,22 @@ const (
 	// instance URI to use.
 	MySQLURIEnv = "TEST_MYSQL_URI"
 
-	defaultTestMySQLURI = "root@tcp(127.0.0.1)"
+	// Note: sql.Open requires the URI to end with a slash.
```

Comment:
Is this true? looks like its rather that the way you use this value within a construction of a path that is then passed to sql.Open that requires a `/`? (I'm referring to L118 where I don't think mysqlURI()+name ends with a `/`, but there may be other uses.)

Reply:
This is true. At the moment some integration tests fail because of this bug (but CI is temporarily unavailable, so we don't see it).

See also line 67, where `mysqlURI()` is passed to `sql.Open` as is. This is the line where currently the fail happens.